util/dvsim/LocalLauncher.py

```diff
@@ -49,6 +52,13 @@ def _do_launch(self):
                      errors="surrogateescape")
             f.write("[Executing]:\n{}\n\n".format(self.deploy.cmd))
             f.flush()
+            if self.deploy.timeout_mins:
+                log.log(VERBOSE, "Job %s timeout set to %d minutes",
+                        self.deploy.name, self.deploy.timeout_mins)
```

Comment:
The nightly CI (IIRC) sets the `--verbose` flag, which will cause this log to be printed 30k times and flood the kokoro log. Consider making this `log.debug()` instead.

Reply:
I may remove it altogether if the kill message can be modified as you suggest.

Reply:
Done